routing/serialize_speed_camera.cpp

```diff
@@ -0,0 +1,45 @@
+#include "routing/serialize_speed_camera.hpp"
+
+#include "coding/pointd_to_pointu.hpp"
+#include "coding/varint.hpp"
+#include "coding/write_to_sink.hpp"
+
+#include "base/assert.hpp"
+
+namespace routing
+{
+uint32_t constexpr SpeedCameraMwmHeader::kLatestVersion;
+
+void SerializeSpeedCamera(FileWriter & writer, uint32_t prevFeatureId, const routing::SpeedCameraMetadata & data)
+{
+  CHECK_EQUAL(data.m_ways.size(), 1, ());
+
+  auto const & way = data.m_ways.back();
+
+  auto featureId = static_cast<uint32_t>(way.m_featureId);
+  CHECK_GREATER_OR_EQUAL(featureId, prevFeatureId, ());
+  featureId -= prevFeatureId;  // delta coding
+  WriteVarUint(writer, featureId);
+  WriteVarUint(writer, way.m_segmentId);
+
+  uint32_t const coef = DoubleToUint32(way.m_coef, 0, 1, 32);
```

Comment:
In line comments pls for 0, 1, 32... Even more. Write please: `uint32_t const coef = DoubleToUint32(way.m_coef, 0.0 /* min */, 1.0 /* max */, 32 /* bits */);`